app/controllers/blog_posts_controller.rb

```diff
@@ -8,11 +8,17 @@
 class BlogPostsController < ApplicationController
   def index
     @blog_posts = Pro::BlogPost.includes(:network).page(blog_posts_page).per(6).all
+    @hero_image = homepage_hero_image
   end
 
   protected
 
   def blog_posts_page
     (params[:page] || 1).to_i
   end
+
+  def homepage_hero_image
+    landing_page = Page::Landing.find_by_slug('')
```

Comment:
I will use a class method for `Page::Landing.home` so that it only returns the one instance, not a scope which would return a collection with only one member.

I will put `homepage_hero_image` into a concern instead of a helper because db calls should be made before the view is rendered, and concerns better isolate to the contexts that need the methods than helpers.